spanner/google/cloud/spanner/streamed.py

```diff
@@ -130,6 +134,7 @@ def consume_next(self):
         self._resume_token = response.resume_token
 
         if self._metadata is None:  # first response
+            # XXX: copy implicit txn ID to snapshot, if present.
```

Comment:
Please remove the `XXX` (either delete or file an issue or change it to a `NOTE:`)